kubernetes/ama-logs.yaml

```diff
@@ -454,6 +454,8 @@ spec:
               value: "false"
             - name: APPMONITORING_OPENTELEMETRYLOGS_PORT
               value: "28331"
+            - name: AZMON_OPENTELEMETRYLOGS_CONTAINER_PORT
```

Comment:
make the yaml changes consistent with AKS yaml with regards to open telemetry feature.